src/component/mxgraph/renderer/CoordinatesTranslator.ts

```diff
@@ -47,9 +47,9 @@ export default class CoordinatesTranslator {
     const model = this.graph.getModel();
     const offset = new mxPoint(0, 0);
 
-    while (cell != null) {
+    while (cell != undefined) {
```

Comment:
**question:** while not using `while(cell)` (same remark for other check using the same pattern)
Checking _falsability_ covers both null and undefined for objects